processor/spanmetricsprocessor/processor.go

```diff
@@ -401,7 +402,8 @@ func (p *processorImp) aggregateMetrics(traces ptrace.Traces) {
 				buildKey(p.keyBuf, serviceName, span, p.dimensions, resourceAttr)
 				key := metricKey(p.keyBuf.String())
 				p.cache(serviceName, span, key, resourceAttr)
-				p.updateHistogram(key, latencyInMilliseconds, span.TraceID(), span.SpanID())
+				asc := uint64(1 << tracestate.GetPValue(span.TraceState()))
```

Comment:
rereading the spec I think I goofed here and we need special handling for PValue of 63 as
```
Zero adjusted count is represented by the special p-value 63, otherwise the p-value is set to the negative base-2 logarithm of sampling probability:
```[
](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/tracestate-probability-sampling.md#p-value)